homeassistant/components/rpi_gpio_pwm/const.py

```diff
@@ -0,0 +1,40 @@
+"""Constants for the rpi_gpio_pwm integration."""
+
+from homeassistant.components.light import (
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR,
+    SUPPORT_TRANSITION,
+)
+
+DOMAIN = "rpi_gpio_pwm"
+
+CONF_LEDS = "leds"
+CONF_NUMBERS = "numbers"
+CONF_DRIVER = "driver"
+CONF_PINS = "pins"
+CONF_PIN = "pin"
+CONF_FREQUENCY = "frequency"
+CONF_INVERT = "invert"
+CONF_STEP = "step"
+
+CONF_DRIVER_GPIO = "gpio"
+CONF_DRIVER_PCA9685 = "pca9685"
+CONF_DRIVER_TYPES = [CONF_DRIVER_GPIO, CONF_DRIVER_PCA9685]
+
+CONF_LED_TYPE_SIMPLE = "simple"
+CONF_LED_TYPE_RGB = "rgb"
+CONF_LED_TYPE_RGBW = "rgbw"
+CONF_LED_TYPES = [CONF_LED_TYPE_SIMPLE, CONF_LED_TYPE_RGB, CONF_LED_TYPE_RGBW]
+
+DEFAULT_BRIGHTNESS = 255
+DEFAULT_COLOR = [0, 0]
+
+SUPPORT_SIMPLE_LED = SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
+SUPPORT_RGB_LED = SUPPORT_BRIGHTNESS | SUPPORT_COLOR | SUPPORT_TRANSITION
```

Comment:
Let's keep the light specific stuff in `light.py`

Reply:
These are constants, so I would expect them in a contstants file, or?

Reply:
Done in commit cecf42dc8f7477c7490f004b1e399c396a15b2ed